cmd/notifier/messenger.go

```diff
@@ -0,0 +1,135 @@
+package main
+
+import (
+	"encoding/json"
+	"errors"
+	"log"
+	"time"
+
+	"github.com/status-im/status-go/geth/node"
+	"github.com/status-im/status-go/messenger"
+)
+
+const (
+	// defaultWorkTime is a work time reported in messages sent to MailServer nodes.
+	defaultWorkTime = 5
+)
+
+var (
+	// ErrInvalidSymKeyID is returned when it fails to get a symmetric key.
+	ErrInvalidSymKeyID = errors.New("invalid symKeyID value")
+)
+
+type BroadcastMsg struct {
+	PubKey string `json:"pubkey"`
```

Comment:
Where is this being used? I'd expect it to be used in `BroadcastAvailability` to pass the pubkey to the clients (which they will use to encrypt their response to us).

Reply:
I'll remove it, it was like this. But that logic was moved to the messenger library. 

Now this will happen through the [NewContactKeyRequest](https://github.com/status-im/status-go/pull/863/files#diff-e0aa9266aa1e17a3ee515e4d8af3f93eR81) as described on the  New status protocol doc:
```
[
  "~#c1",
  [
    "0xc2a38488c043b7c0d138e922887bfcc2966c7084....",
    "0x2d215ad8",
    [
      "~#c2",
      [
        "my name",
        "data:image/png;base64,base64-representation-of-profile-image...",
        "e5ac45596d8be7aabea51c0dbbac34.....",
        "dFLRZwPx3yQ:APA91bH_7yjq47N5HY0ciKs58HW3VbJ9...”
      ]
    ]
  ]
]
```